realm/realm-library/src/objectServer/java/io/realm/mongodb/App.java

```diff
@@ -511,16 +514,34 @@ public AppConfiguration getConfiguration() {
     }
 
     /**
-     * Exposed for testing.
+     * Creates a request for a streaming function
      *
+     * @param user         that requests the execution
+     * @param functionName name of the function
+     * @param bsonArgs     function arguments as a {@link BsonArray}
+     * @param serviceName  service that will handle the function
+     * @return {@link io.realm.internal.objectstore.OsJavaNetworkTransport.Request}
+     */
+    OsJavaNetworkTransport.Request makeStreamingRequest(User user,
```

Comment:
Not 100% sure why this is in the App class even when package protected? It feels a bit out of place?

Reply:
I placed `makeStreamingRequest` there because it is an actual method of the obj-store app class. I thought it was the right pattern to reflect native methods on Java.